fdbserver/workloads/TenantManagementWorkload.actor.cpp

```diff
@@ -1284,11 +1284,32 @@ struct TenantManagementWorkload : TestWorkload {
 
 		loop {
 			try {
+				state Optional<TenantGroupName> tGroup = self->chooseTenantGroup(false, false);
 				if (self->useMetacluster) {
 					state std::vector<std::pair<TenantName, metacluster::MetaclusterTenantMapEntry>> metaTenants =
 					    wait(metacluster::listTenantMetadata(self->mvDb, beginTenant, endTenant, limit));
 					verifyTenantList<metacluster::MetaclusterTenantMapEntry>(
 					    self, metaTenants, limit, beginTenant, endTenant);
+
+					// Get all tenants in the specified tenant group
+					state std::vector<std::pair<TenantName, metacluster::MetaclusterTenantMapEntry>>
```

Comment:
Rather than a run a second, minimally parameterized list for the group, I think it would be preferable to add a tenant group filter to the original list query and verify it. For that to work, you'll need to update `verifyTenantList` to skip local tenants that aren't in the group.